README.md

```diff
@@ -291,6 +291,14 @@ try {
 }
 ```
 
+To enable caching for the requests, one can add `expiresIn` to the requests configuration object. The values are in seconds.
+```
+// cache is valid for 30 minutes
+const requestsConfig = {
+  expiresIn: 1800,
```

Comment:
Looking at this interface description, it is not clear the `expiresIn` enables caching (or that it is connected to cache at all, for that matter).

Maybe we should change the interface so that `cache` key must be set for caching to be enabled?

Also, maybe we should rename `expiresIn` to `cacheExpiresIn`, `cacheLifetime` or similar? And set the default value for it when caching is enabled?

Note that this would be a breaking change if we were to change it later, so it pays to put some effort in designing a better interface now...

Reply:
The interface has changed to following and future elements like `target` will also move in to the object
```
  cache: {
  expiresIn: 1800,
  }
};
```